### PR TITLE
Lack of logging level verbosity argument

### PR DESCRIPTION
**Pull request by r8d8**
Added cli args for log level & added logging prints
Related to #132

### CODE REVIEW

File: emerald-cli/usage.txt [hunk 1]
```diff
@@ -6,16 +6,16 @@ Usage:
...
-      --base-path <path>                    Base directory path, if omitted default os-specific value will be used:
-                                              + Mac OS X: ~/Library/Emerald
-                                              + Linux: ~/.emerald
-                                              + Windows: %USERDIR%\.emerald
-      --security-level <normal|high|ultra>  Level of security for cryptographic operations [default: normal]
-  -c, --chain <mainnet|testnet>             Chain name [default: mainnet]
-      --chain-id <id>                       Chain id number [default: 61]
+  -h, --help                                  Show this message
+  -V, --version                               Show current version
+  -q, --quiet                                 Switch off logging
```
Comment (dulanov) [inline emerald-cli/usage.txt]: Not switch off completely, only errors.

File: emerald-cli/usage.txt [hunk 2]
```diff
@@ -6,16 +6,16 @@ Usage:
...
+  -v, --verbose                               Verbose logging
```
Comment (dulanov) [inline emerald-cli/usage.txt]: Not logging, probably the word `output` will be better.

File: emerald-cli/usage.txt [hunk 3]
```diff
@@ -6,16 +6,16 @@ Usage:
...
+      --host <host>                           Listen host [default: 127.0.0.1]
+      --port <port>                           Listen port [default: 1920]
+      --base-path <path>                      Base directory path, if omitted default os-specific value will be used:
+                                                  + Mac OS X: ~/Library/Emerald
+                                                  + Linux: ~/.emerald
+                                                  + Windows: %USERDIR%\.emerald
+      --security-level <normal|high|ultra>    Level of security for cryptographic operations [default: normal]
+  -c, --chain <mainnet|testnet>               Chain name [default: mainnet]
+      --chain-id <id>                         Chain id number [default: 61]
```
Comment (dulanov) [inline emerald-cli/usage.txt]: I think we should delete default number here, for clarity. By default, there is only `--chain`, not `--chain-id`.

Comment (r8d8) [inline emerald-cli/usage.txt]: removed in fix for #157

**Pull request merged**